RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate REST API - emitTyping to Typescript

## Proposed changes

Migrate REST API - emitTyping to Typescript

## Issue(s)	


## How to test or reproduce


- Login with 2 different accounts in 2 different devices
- Then, inside the same room, start to typing and the other user should see the information that you are typing

## Screenshots

## Types of changes




## Checklist



## Further comments
